File: Easy/Python/1217. Minimum Cost to Move Chips to The Same Position.py
class Solution(object):
    def minCostToMoveChips(self, position):
        even = 0
        odd = 0

        for pos in position:
            if pos % 2 == 0:
                even += 1
            else:
                odd += 1
        
        return min(even, odd)

File: Easy/JavaScript/1217. Minimum Cost to Move Chips to The Same Position.js
var minCostToMoveChips = function(position) {
    let even = 0;
    let odd = 0;

    for(let pos of position){
        if(pos % 2 == 0)
            even++;
        else
            odd++;
    }

    return Math.min(even, odd);
};
